src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/customer.html.twig

```diff
@@ -69,10 +72,12 @@
           </a>
         </p>
 
-        <p class="mb-1">
-          <strong>{{ 'Account registered:'|trans({}, 'Admin.Orderscustomers.Feature') }}</strong>
-        </p>
-        <p>{{ orderForViewing.customer.accountRegistrationDate|date("m/d/Y H:i:s") }}</p>
+        {%  if orderForViewing.customer.isGuest is same as(false) %}
```

Comment:
@matthieu-rolland Why use `is same as(false)` here but not `is same as(true)` [line 47](https://github.com/PrestaShop/PrestaShop/pull/17985/files#diff-2dc0ae8313c91a44f9ea21306a80f03bR47) ?

Reply:
Because the above code should be hidden if custom.isGuest (you can not see a registered time on a guest customer)